pico_w/mqtt/mqtt_client/mqtt_client.c

```diff
@@ -0,0 +1,161 @@
+//
+// Created by elliot on 25/05/24.
```

Comment:
Could you add a comment clarifying the license that this code is released under please?